assets/app/styles/_create.less

```diff
@@ -0,0 +1,225 @@
+//
+// Create App
```

Comment:
Yeah, easy to do with a general name like that.  Doesn't encourage decomposition.